### ISSUE TITLE
Need help , there is nowhere mentioned how to add a student as a user.

### ISSUE THREAD

**Issue by Ishaan28malik** [jupyter/nbgrader#1449]
### I added a new user in my Jupyerhub now I want to give that user the student rights
* How to share the assignments with the students so that they can access it directly over the jupyterHub ?
* Or create a student account with the rights to access the assignments ?

**Comment by perllaghu** [comment]
Do you mean "add a user to the system so they can log in" or "add an existing login as a student"?

The former is a jupyterhub issue, rather than nbgrader.

The latter is discussed in the documentation [I'm in the wrong place to get a link] relating to the configuration.

**Comment by Ishaan28malik** [comment]
You got the issue a bit right . I logged in as a professor and now I am able to preview my assignment as a student but with my account only. Now how to create a student account and login using that and then attempt the assignment in the assignments section ? 

The way @jhamrick did in her talk at `Scipy 17`

**Comment by perllaghu** [comment]
They are just two different users in jupyterhub.

I don't recall the presentation.... but having two different users logged in is no problem for the hub (you just need to be clever with browsers if you're using the Web interface)

**Comment by Ishaan28malik** [comment]
I added the user in jupyterhub revoking the admin rights , what I am asking is that how can that user access the assignments ? The user I created is having access to Formgrader and assignments tab but there is no data inside them as they were in the teacher's account.

**Comment by perllaghu** [comment]
If you've got two users in the same 'hub, both running notebooks that have `formgrader` and `assignments` tab.... the next question is _have you set the `course_id` for both users_ ?
 https://nbgrader.readthedocs.io/en/0.6.x/user_guide/philosophy.html#configuration-files

**Comment by Ishaan28malik** [comment]
Yeah I checked that but that's how we can add a new collaborator in our `nbgrader` or similar to how we add a user as an `admin` inside the jupyterHub .

**Comment by guillermouberomero** [comment]
Hi, I had the same problem and i achieved to fix it.

Check that you`ve set an nbgrader_config.py in a global path (see which one is valid for your system with jupyter--paths), in which you specify:

        c.Exchange.root = "/tmp/exchange"

Note that this directory MUST exist before running nbgrader, and it must be readable and writable by all users. To configure it:

       # remove existing directory, so we can start fresh for demo purposes
       rm -rf /tmp/exchange
       # create the exchange directory, with write permissions for everyone
       mkdir /tmp/exchange
       chmod ugo+rw /tmp/exchange

Then, I don`t know if it's necessary but i have another nbgrader_config.py in the root directory of the course, another one inside the course directory, and another one for each JupyterHub user indicating the root directory of the course (see how to create an example nbgrader_config.py).

After that, to link a JupyterHub user with a course student, make sure the Student ID is the same as the JupyterHub username.

I hope it`ll help you
